editor/src/components/canvas/canvas-utils.ts

```diff
@@ -1471,6 +1475,10 @@ export function produceResizeCanvasTransientState(
   }
 }
 
+export function isTargetPropertyHorizontal(edgePosition: EdgePosition): boolean {
+  return edgePosition.x !== 0.5
```

Comment:
These comparisons to `0.5` seem dangerous to me. We have a function `numbersEqual` in `math-utils.ts` for these kinds of comparisons

Reply:
I don't think that should be a problem as these are all constant values anyway I believe everywhere they're defined and we're comparing these values in a few places like this.